src/runtime-rs/crates/resource/src/cgroups/utils.rs

```diff
@@ -14,3 +25,20 @@
 pub(crate) fn gen_overhead_path(path: &str) -> String {
     format!("kata_overhead/{}", path.trim_start_matches('/'))
 }
+
+pub(crate) fn rename_cgroup_path(path: &str) -> String {
+    let cgroup_path = if path.is_empty() {
+        Path::new(DEFAULT_RESOURCE_CONTROLLER_ID)
+    } else {
+        Path::new(path)
+    };
+
+    let cgroup_path_dir = cgroup_path.parent().unwrap().to_string_lossy().to_string();
+    let cgroup_path_name = format!(
```

Comment:
cgroup_path_name could be invalid when cgroup_path = Path::new(DEFAULT_RESOURCE_CONTROLLER_ID)

Reply:
Thanks for your review, I fixed it.